www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -530,9 +530,13 @@
           </check001>
         </Constraints>
       </message>
-      <identifier type="IntegerField">
+      <identifier type="TextField">
```

Comment:
We also have a CSV type which can be used width a select multiple if you want to use that in the frontend. In the Backend you will still get the CSV. This is up to you.